src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -38,6 +30,41 @@ function crosses_vertex(t::Scaled{C,Prod{Op}}, g::AbstractGraph, v) where {C}
   return v ∈ span(t, g)
 end
 
+# map all vertices `w` of `g` except for `v` to the incident edge of `v`
+# which lies in edge_path(g,w,v)
+function map_vertices_to_incident_edges(g::AbstractGraph, v, es)
```

Comment:
I don't really understand what this function is doing. In what way is it a "map"?

Reply:
It creates a `Dict` from vertices to incident edges.

Reply:
I renamed it to `vertices_to_incident_edges_dict` and added some comments to explain what it's doing.

Reply:
This has been simplified and moved into `ttn_svd` function body.